src/node-backend/routes/classroom.js

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import * as classroomController from "../controller/classroom-controller.js";
+
+import { JoiningStudent } from "../models/domain-objects.js";
+export const router = express.Router();
+
+router.post('JoiningStudent', async function (req, res) {
+    try {
+        const {username} = req.body;
+        
+        res.status(200).json(result);
+    } catch (error) {
+        res.status(409).json({ message: error.message });
+    }
+});
+
+router.post('saveStudents', async function (req, res) {

```

Comment:
use the comments I added (for example here: https://github.com/WomenPlusPlus/deploy-impact-22-wikimini-b/blob/development/src/node-backend/routes/classrooms.js) to know which data you need to receive from the frontend in the req.body :) you don't need to create the studentList for example, this is really just to define how frontend and backend will communicate, it doesn't actually have to do anything :)